### ISSUE TITLE
Sign in options using firebase

### ISSUE THREAD

**Issue by Navaneeth-R-Krishnan** [Yeasir0032/Discord-Clone#8]
I'd like to add signin options using firebase authentication, kindly assign me this under GSSoC '24.

**Comment by Yeasir0032** [comment]
Could you use Supabase (A firebase Alternative) for authentication. This will replace ClerkJs. It will be better with Supabase as this repo currently depends upon supabase-js

**Comment by Navaneeth-R-Krishnan** [comment]
Yes, I would be able to do that, just a quick read through supabase and I can work on it and contribute efficiently, kindly assign me this issue.

**Comment by Yeasir0032** [comment]
Okay I am assigning this issue to you.
Here are some steps you can follow :- 
- Read about supabase authentication system.
- Understand how supabase is used in this project. You can look into page.tsx files.
- Replace ClerkJs functionality.
- And make sure to remove clerk js from package.json

**Comment by IP80808080** [comment]
👋 Hello! I'd be happy to help with this issue. Could you please assign it to me? Additionally, could you please label it for the appropriate level? Thank you! 🚀

**Comment by PayalKumari10** [comment]
@Yeasir0032   i want to work on this issues, can you assign me ?

**Comment by Yeasir0032** [comment]
Sorry @PayalKumari10 and @IP80808080 
I have assigned this issue to @Navaneeth-R-Krishnan , but don't be upset I will add more issues shortly, you can work upon those.

**Comment by Yeasir0032** [comment]
Hey @Navaneeth-R-Krishnan 
How far you have reached?

**Comment by codebreaker3008** [comment]
Can I do this if not completed??

**Comment by codebreaker3008** [comment]
@Navaneeth-R-Krishnan how far did you get, connect with me via mail"ak7085330@gmail.com" and we can help out each other of you want to and sort this out.

**Comment by codebreaker3008** [comment]
@Yeasir0032 could you explain a bit whats the current situation on this ?

**Comment by Yeasir0032** [comment]
@codebreaker3008 
Currently it uses clerkjs 
You have to learn authentication using supabase auth.
You should replace ClerkJs completely.

**Comment by omsai11** [comment]
@Navaneeth-R-Krishnan Would you please tell us the status on this issue?

**Comment by Yeasir0032** [comment]
How is it going @codebreaker3008 ?

**Comment by Aditya19110** [comment]
Is the issue completed if not i can help on doing the signin option using superbase auth please assign this task to me @Yeasir0032

**Comment by PayalKumari10** [comment]
Hello @Aditya19110 
Please read the table-setup.md file and learn.md file before working.

**Comment by PayalKumari10** [comment]
Hey @Aditya19110 ,  Is there any update on this issue?
